### ISSUE TITLE
KeyCloak does not function under default install method

### ISSUE THREAD

**Issue by samzph** [camunda/camunda-platform-helm#315]
Installing with no parameters as [in the guide](https://docs.camunda.io/docs/self-managed/platform-deployment/kubernetes-helm/#how-to-install-camunda-8-using-helm) seems to fail due to Keycloak not knowing the randomly generated password for the `bn_keycloak` user. 

**The Issue**:

`$ kubectl logs -n camunda-ee camunda-ee-postgresql-0`
```shell
postgresql 03:06:42.74 INFO  ==> ** Starting PostgreSQL **
2022-05-11 03:06:42.763 GMT [1] LOG:  pgaudit extension initialized
2022-05-11 03:06:42.772 GMT [1] LOG:  starting PostgreSQL 14.2 on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
2022-05-11 03:06:42.772 GMT [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2022-05-11 03:06:42.773 GMT [1] LOG:  listening on IPv6 address "::", port 5432
2022-05-11 03:06:42.777 GMT [1] LOG:  listening on Unix socket "/tmp/.s.PGSQL.5432"
2022-05-11 03:06:42.783 GMT [92] LOG:  database system was shut down at 2022-05-11 03:05:50 GMT
2022-05-11 03:06:42.792 GMT [1] LOG:  database system is ready to accept connections
2022-05-11 03:07:35.743 GMT [160] FATAL:  password authentication failed for user "bn_keycloak"
2022-05-11 03:07:35.743 GMT [160] DETAIL:  Connection matched pg_hba.conf line 1: "host     all             all             0.0.0.0/0               md5"
```

**Steps to reproduce**:
```shell
helm repo add camunda https://helm.camunda.io
helm repo update
kubectl create ns camunda-8
helm install camunda-8 -n camunda-8 camunda/camunda-platform
```

**Attempted Solutions**:
For the past few hours I've been following along on #127 trying to do this manually using `--set` values, but I'm not having luck getting it to populate all the way down to the postgres chart nested within the keycloak chart nested within the identy chart (I can keep going :)).

Basically it seems the current issue is that there is no logic in place to populate the randomly generated by defauly `postgres-password` and `password` from the `<release-name>-postgresql` secret that is created at deployment. So KeyCloak has no idea how to authenticate to postgres.

**Comment by Zelldon** [comment]
Hey @samzph 

thanks for your report. I have several follow up question please see below:

* Could you please elaborate a bit on the impact you see with that? Is it just the logging or are you not able to login or anything? 
* Which version of the Helm charts you are using? 
* Could you please share the values.yaml file, otherwise it might be hard to reproduce. I'm wondering that you mention in the last paragraph that you use a values.yaml file and in steps to reproduce there is no reference to it.
* Did you an upgrade of the helm charts? Or reinstalled them? It might be that you reusung old PVC's

**Comment by Zelldon** [comment]
Happy that it works now @samzph thanks for sharing :+1:

Just that you're aware if you use **upgrade** you need to set the passwords via --set or via your values file on the second attempt, otherwise you get issues with regenerated secrets. You can read about it here 

https://stage.docs.camunda.io/docs/self-managed/platform-deployment/kubernetes-helm/#upgrading-from-one-helm-release-to-another
